test/connectors/MessageConnectorSpec.scala

```diff
@@ -85,6 +86,7 @@ class MessageConnectorSpec
             .withHeader("X-Message-Type", equalTo(messageType.toString))
             .withHeader("X-Message-Sender", equalTo(messageSender))
             .withRequestBody(matchingXPath("/transitRequest"))
+            .withHeader("channel", equalTo(web.toString))
```

Comment:
@RobertLaverick If possible can we use property-based testing to pick either API or web instead of hardcoding

Reply:
sure, I'll get that done and push it ready to merge in the morning